diff --git a/src/dftbp/extlibs/tblite.F90 b/src/dftbp/extlibs/tblite.F90
index 044a0f5fcf..931b63bc07 100644
--- a/src/dftbp/extlibs/tblite.F90
+++ b/src/dftbp/extlibs/tblite.F90
@@ -431,16 +431,22 @@ subroutine getCalculator(method, mol, calc)
     !> Parametrisation data
     type(xtb_calculator), intent(out) :: calc
 
+    type(error_type), allocatable :: err
+
     select case(method)
     case default
       call error("Unknown method selector")
     case(tbliteMethod%gfn2xtb)
-      call new_gfn2_calculator(calc, mol)
+      call new_gfn2_calculator(calc, mol, err)
     case(tbliteMethod%gfn1xtb)
-      call new_gfn1_calculator(calc, mol)
+      call new_gfn1_calculator(calc, mol, err)
     case(tbliteMethod%ipea1xtb)
-      call new_ipea1_calculator(calc, mol)
+      call new_ipea1_calculator(calc, mol, err)
     end select
+    if (allocated(err)) then
+      call error(err%message)
+    end if
+
   end subroutine getCalculator
 #:endif
 
